pkg/oc/cli/importimage/importimage.go

```diff
@@ -384,39 +413,55 @@ func (o *ImportImageOptions) importTag(stream *imageapi.ImageStream) (*imageapi.
 		existing.Generation = &zero
 
 	}
-	stream.Spec.Tags[tag] = *existing
```

Comment:
@deads2k imageapiv1 switches an imagestream's `spec.tags` field from a `map[string]TagReference` to a slice of TagReferences. Not sure how to handle tag lookups in the command

Reply:
> @deads2k imageapiv1 switches an imagestream's spec.tags field from a map[string]TagReference to a slice of TagReferences. Not sure how to handle tag lookups in the command

You can't just iterate over the list?  There shouldn't be any conflicting names.

Reply:
I did not want to assume that a `ReferenceTag.Name` value was the same as the value used for a key in the map that was used in the internal api. Latest changes to this PR make use of the imageapi.SplitImageStreamTag helper: https://github.com/openshift/origin/blob/6f8c120068ac61b5c1728b9c2386a8d882bcf501/pkg/image/apis/image/helper.go#L161